app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_save :update_likes
+
+  def update_likes
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ OPTIONAL ] Great job for using one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_likes method every time a new like has been saved to database, but please I recommend making the update_likes method private ✨🚀🔥